php/elFinderVolumeDriver.class.php

```diff
@@ -1182,6 +1182,22 @@ public function open($hash) {
 	}
 	
 	/**
+	 * md5sum from file
+	 *
+	 * @param  string   file hash
+	 * @return string
+	 * @author gabriel.rota@gmail.com
+	 **/
+	public function md5sum($hash) {
+		if (($file = $this->file($hash)) == false
+		|| $file['mime'] == 'directory') {
+			return false;
+		}
+		
+		return $this->_md5sum($this->decode($hash), 'rb');
```

Comment:
- What is the second argument for?
- Won't this explode for file systems other than the local FS? I suggest adding a stub or abstract method `elFinderVolumeDriver::_md5sum()`.